src/main/java/com/exasol/projectkeeper/validators/DeletedFilesValidator.java

```diff
@@ -38,19 +38,21 @@ public DeletedFilesValidator(final Path projectDirectory, final ExcludedFilesMat
     }
 
     @Override
-    public Validator validate(final Consumer<ValidationFinding> findingConsumer) {
-        FILES_THAT_MUST_NOT_EXIST.forEach((fileThatMustNotExist, reason) -> {
+    public List<ValidationFinding> validate() {
+        return FILES_THAT_MUST_NOT_EXIST.entrySet().stream().map((entry) -> {
```

Comment:
I would rename this to `FILES_TO_DELETE`

Reply:
well, the env var just contains a list of files that must not exist in a project. Usually, they don't exist. So we can't delete them.
That's why I called it `FILES_THAT_MUST_NOT_EXIST` since it reflects the rule this is valiudated.